### ISSUE TITLE
PHP open/close tags on new lines should not apply for PHP inside HTML

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#423]
Code:
```php
<?php
// something
?>
<table>
	<tr><td>Usage: <strong><?php echo round($bin_packer->percentsUsed(), 2) . '%'; ?></strong></td><td><a href="<?php echo $this->getLink('up'); ?>">UP</a></td><td>Zoom: <strong><?php echo $this->sizeCoefficient; ?></strong></td></tr>
	<tr><td><a href="<?php echo $this->getLink('left'); ?>">LEFT</a></td><td><img style="border: 1px solid black; padding: 2px;" src="<?php echo $base_url . '/system/object.png'; ?>" alt=""/></td><td><a href="<?php echo $this->getLink('right'); ?>">RIGHT</a></td></tr>
	<tr><td>&nbsp;</td><td><a href="<?php echo $this->getLink('down'); ?>">DOWN</a></td><td>&nbsp;</td></tr>
</table>
```
For code above I'm getting following errors:
* The opening PHP tag must be the first content in the file (Generic.PHP.CharacterBeforePHPOpeningTag.Found)
* Each PHP statement must be on a line by itself (Generic.Formatting.DisallowMultipleStatements.SameLine)

I'm not 100% sure, but for cases, when `<?php` and `?>` are located on same line at least 1st error (`Generic.PHP.CharacterBeforePHPOpeningTag.Found`) should not be reported/fixed.

**Comment by gsherwood** [comment]
Both those sniffs were written to disallow the sample code you've provided, but I agree that they work better when they allow that code instead. I've made 2 commits to let this sample code pass.

**Comment by aik099** [comment]
Great. Thanks.
